### ISSUE TITLE
Audio Muting for Syfy

### ISSUE THREAD

**Issue by richardfrost** [FrostCo/AdvancedProfanityFilter#656]
### :movie_camera: [Syfy](https://www.syfy.com)

This issue is here for sharing updates and reporting issues related to this supported site. Feel free to leave a comment if you have any feedback to share.

**Comment by richardfrost** [comment]
Syfy has updated their site and requires an update to keep filtering.

```json
{
  "www.syfy.com": {
    "mode": "element",
    "className": "jw-text-track-container",
    "subtitleSelector": "div.jw-text-track-cue",
    "tagName": "DIV"
  }
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

For best results, make sure to remove any custom config used for testing like this once it has been included in the official update.

**Comment by richardfrost** [comment]
This is now included in version [4.1.0](https://github.com/FrostCo/AdvancedProfanityFilter/releases/tag/4.1.0). If you previously used the config shared in this issue you should now be able to remove it.
